### PR TITLE
Update flutter and libs

### PR DESCRIPTION
**Pull request by 0legg**
Flutter tends to complain of being older than NOW. Oh, I'm too old for modern development, but who cares.

### CODE REVIEW

File: .flutter-plugins [hunk 1]
```diff
@@ -1,2 +1,2 @@
-share=/Users/jbarr/.pub-cache/hosted/pub.dartlang.org/share-0.4.0/
-url_launcher=/Users/jbarr/.pub-cache/hosted/pub.dartlang.org/url_launcher-3.0.0/
+share=/Users/olegg/.pub-cache/hosted/pub.dartlang.org/share-0.5.2/
+url_launcher=/Users/olegg/.pub-cache/hosted/pub.dartlang.org/url_launcher-3.0.3/
```
Comment (0legg) [inline .flutter-plugins:2]: Should go to .gitignore actually

File: ios/Flutter/flutter_assets/AssetManifest.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"images/icons/github.png":["images/icons/github.png"],"images/icons/imgur.png":["images/icons/imgur.png"],"images/icons/reddit.png":["images/icons/reddit.png"],"images/icons/ycombinator.png":["images/icons/ycombinator.png"]}
```
Comment (0legg) [inline ios/Flutter/flutter_assets/AssetManifest.json:1]: Assets should be .gitignored as well

File: .flutter-plugins (hunk 1 above)
Comment (jbarr21) [reply .flutter-plugins:2]: I will update the .gitignore in a followup PR from a more modern project

**Review by jbarr21 — APPROVED** [review]

**Comment by jbarr21** [comment]
Nice, thanks!

**Pull request merged**